extensions/amp-analytics/0.1/variables.js

```diff
@@ -220,7 +220,7 @@ export class VariableService {
         // Expand string values further.
         p = this.expandTemplate(raw,
             new ExpansionOptions(options.vars, options.iterations - 1,
-                options.noEncode));
+                true /* noEncode */));
```

Comment:
`noEncode=true` is tested [here](https://github.com/ampproject/amphtml/pull/10565/files#diff-7713b5884d853d9e661d5d4768037b22R70)